proto/pocket/shared/service.proto

```diff
@@ -32,7 +28,7 @@ message ApplicationServiceConfig {
 
 // SupplierServiceConfig holds the service configuration the supplier stakes for
 message SupplierServiceConfig {
-    ServiceId service_id = 1; // Unique and semantic identifier for the service
+    Service service = 1; // The Service for which the supplier is configured for
```

Comment:
```suggestion
    Service service = 1; // The Service for which the supplier is configured
```

Reply:
Didn't notice this so will tend to it in #123